wdl-engine/tests/inputs/missing-file/error.txt

```diff
@@ -1,4 +1,5 @@
-failed to open input file `tests/inputs/missing-file/inputs.json`
+failed to parse input file `tests/inputs/missing-file/inputs.json`
```

Comment:
This error changed because I moved a `context` call around to accommodate identifier key verification earlier in the process. I felt it was a fine side effect, even though it's a bit redundant.